internal/pkg/aws/ecr/ecr.go

```diff
@@ -43,36 +43,28 @@ func New(s *session.Session) ECR {
 	}
 }
 
-// Auth represent basic authentication credentials.
-type Auth struct {
-	Username string
-	Password string
-}
-
-// GetECRAuth returns the basic authentication credentials needed to push images.
-func (c ECR) GetECRAuth() (Auth, error) {
+// Auth returns the basic authentication credentials needed to push images.
+func (c ECR) Auth() (string, string, error) {
```

Comment:
what do you think of this?
```suggestion
func (c ECR) Auth() (username string, password string, err error) {
```

one benefit for this is you don't need https://github.com/aws/copilot-cli/pull/1174/files#diff-af397e6d8c2e47689a45b17a28f0b4cbR62. And the second benefit is when using this method it is easier to understand what they'll expect for each output.